snaps/reference/cli/options.md

```diff
@@ -1,14 +1,31 @@
 ---
 sidebar_label: Options
 sidebar_position: 1
+description: See the Snaps CLI options reference.
 ---
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 # Snaps command line options
 
-## b, bundle
+This reference describes the syntax of the Snaps command line interface (CLI) options.
+
+## Specify options
+
+You can specify options:
```

Comment:
Not sure if this format is necessary. Maybe... "You can specify options either in the configuration file (...) or on the command line using ...